.deploy/app.yaml

```diff
@@ -25,9 +25,9 @@ spec:
       valuesFiles:
         - values.yaml
   valuesFrom:
-  - kind: ConfigMap
-    name: flux-values
-    valuesKey: values.yaml
+    - kind: ConfigMap
```

Comment:
Is the extra indentation intentional?

Reply:
no, it's yq doing that because probably i never linted the original file